docs/cli_README.md

```diff
@@ -0,0 +1,45 @@
+## CRDA CLI
+
+#### CRDA CLI is a CLI Tool to interact with CRDA Platform. 
+
+### Installation:
+- Select, Download and Install latest binary from [Releases](https://github.com/fabric8-analytics/cli-tools/releases)
+
+### Usage:
+Executable supports following commands:
+
+- `crda auth`: This command is used to enable user to Authenticate with CRDA Server. This command generates and saves Auth Token in `$HOME/.crda/config.yaml`
+    Supported Flags:
+    * `--synk-token` (string) (OPTIONAL): Can be obtained from [here](https://app.snyk.io/login?utm_campaign=Code-Ready-Analytics-2020&utm_source=code_ready&code_ready=FF1B53D9-57BE-4613-96D7-1D06066C38C9). If not set, Freemium a/c with limited functionality will be created.
+    * `--auth-token` (string) (OPTIONAL): 3Scale Gateway Authentication token. Needed if you wish to access Auth APIs hosted at plateform other than CRDA.
```

Comment:
Is there an option to provide an auth token?

Reply:
Yes, Via this flag

Reply:
Otherwise, we have a default preset value of Production token